src/lib/artsy_jwt.ts

```diff
@@ -0,0 +1,10 @@
+import jwt from "jwt-simple"
+import config from "config"
+const { HMAC_SECRET } = config
+
+export const decodeArtsyJWT = token => {
```

Comment:
Adding the type here also means the call-site is checked.

```suggestion
export const decodeArtsyJWT = (token: string) => {
```